support-frontend/assets/helpers/user/user.js

```diff
@@ -130,11 +130,11 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
     dispatch(setPostDeploymentTestUser(true));
   }
 
-  if (getCookie('gu_recurring_contributor') === 'true') {
+  if (getCookie('gu_recurring_contributor')) {
```

Comment:
is this a change in behaviour? What if it's `'false'`?

Reply:
If it's false, then the code inside the block won't execute. I thought `===  true` was unnecessary - it's not a change in the behaviour

Reply:
the comparison was against a string, not a boolean. So if the cookie value is `'false'` then I'd expect the expression to be true?

Reply:
ohhh okay- got it

Reply:
is there another possible value for this cookie? the string and bool thing is confusing

Reply:
the value of a cookie is always a string, so we have to resolve it to a boolean.
It's set based on the response from members-data-api [here](https://github.com/guardian/frontend/blob/3337a586d533cd981f2023773c137bbbb23d1e35/static/src/javascripts/projects/common/modules/commercial/user-features.js#L64)

Reply:
got it - reverting now